src/main/java/randoop/main/CloneVisitor.java

```diff
@@ -0,0 +1,1770 @@
+/*
+ * Copyright (C) 2007-2010 Julio Vilmar Gesser.
+ * Copyright (C) 2011, 2013-2015 The JavaParser Team.
+ *
+ * This file is part of JavaParser.
```

Comment:
Is this really part of JavaParser?  Or was it copied from a file that is part of JavaParser?  Should it be contributed back to JavaParser?

Reply:
This file was directly copied from JavaParser 2.4.0 and so it's now quite outdated. However, it's the latest version which is compatible with the rest of the version of JavaParser which we are using. Should I leave the copyright information?

Reply:
If this file is part of JavaParser 2.4.0, then why does it need to be included in source code form?  Why can's Randoop just use the JavaParser jar file and get it from there?  Can you explain that?  If the file is copied, you should say where it was copied from and whether it has been changed (and how a reader can obtain the diffs).  Definitely don't remove copyright information.